.github/workflows/change-install-version.yaml

```diff
@@ -0,0 +1,31 @@
+name: Change the install version to match the latest tagged version
+on:
+  release:
+    types: [published]
+
+jobs:
+  # Update install version
+  update-install-version:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Get version
+      run: |
+        tarref=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
+        # Strip "v" prefix from tagref
+        if [[ "${{ github.ref }}" == "refs/tags/"* ]]; then
+          echo "VERSION=$(echo $tarref | sed -e 's/^v//')" >> $GITHUB_ENV
+        elif [[ "${{ github.ref }}" == *"master" ]]; then
+          echo "VERSION=latest" >> $GITHUB_ENV
+        fi
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: pip3 install -r ./.github/workflows/requirements.txt
+    - name: Edit install version
+      run: python3 ./.github/workflows/change-install-version.py ${{ env.VERSION }}
+    - uses: EndBug/add-and-commit@v7
```

Comment:
Do we want to just commit? Or do we want to create PR as well?

Reply:
I think committing is less abrasive. If we were to go the other route, then we would need to wait for a workflow and accept a PR after every new release. I think we go with commits for now and we can change it in the future if the requirements are different.

Reply:
This isn't just commit; it is push as well. Any opinions @sriumcp and @huang195 ?